idptest/settings.py

```diff
@@ -113,21 +120,24 @@
 demoSpConfig = {
     'acs_url': 'http://127.0.0.1:9000/sp/acs/',
     'processor': 'saml2idp.demo.Processor',
-    'links': [ # a list of (resource, pattern) tuples, or a {resource: pattern} dict
-        #NOTE: This should still work, due to the "simple" 'login_init' URL in urls.py:
-        #TEST BY BROWSING TO: http://127.0.0.1:8000/sp/test/
+    'links': [
+        # a list of (resource, pattern) tuples, or a {resource: pattern} dict
+        # NOTE: This should still work, due to the "simple" 'login_init' URL in
+        # urls.py:
+        # TEST BY BROWSING TO: http://127.0.0.1:8000/sp/test/
         ('deeplink', 'http://127.0.0.1:9000/sp/%s/'),
-        # The following are "new" deeplink mappings that let you specify more than one capture group:
+        # The following are "new" deeplink mappings that let you specify more
+        # than one capture group:
         # This is equivalent to the above, using the 'new' deeplink mapping:
-        #TEST BY BROWSING TO: http://127.0.0.1:8000/sp/test/
+        # TEST BY BROWSING TO: http://127.0.0.1:8000/sp/test/
         (r'deeplink/(?P<target>\w+)', 'http://127.0.0.1:9000/sp/%(target)s/'),
         # Using two capture groups:
-        #TEST BY BROWSING TO: http://127.0.0.1:8000/sp/test/
+        # TEST BY BROWSING TO: http://127.0.0.1:8000/sp/test/
         (r'deeplink/(?P<target>\w+)/(?P<page>\w+)', 'http://127.0.0.1:9000/%(target)s/%(page)s/'),
         # Deeplink to a resource that requires query parameters:
-        #NOTE: In the pattern, always use %(variable)s, because the captured
+        # NOTE: In the pattern, always use %(variable)s, because the captured
         # parameters will always be in unicode.
-        #TEST BY BROWSING TO: http://127.0.0.1:8000/sp/test/123/
+        # TEST BY BROWSING TO: http://127.0.0.1:8000/sp/test/123/
```

Comment:
tiny thingy, the comment talks about port 8000 but in code we use 9000

Reply:
Good catch!